File: 240612/999 또는 -999/999-or-999.py
n_lst = list(map(int, input().split()))

n_lst = n_lst[:-1]
# print(max(n_lst), min(n_lst))

min_cst, max_cst = n_lst[0], n_lst[0]
for n in n_lst[1:]:
    if n < min_cst:
        min_cst = n
    elif n > max_cst:
        max_cst = n

print(max_cst, min_cst)

File: 251119/999 또는 -999/999-or-999.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main() 
{
    // Please write your code here.
    int min_cst = 999, max_cst = 0;
    vector<int> nums(100);

    for(int i = 0; i < 100; i++)
    {
        cin >> nums[i];
        if(nums[i] == 999 || nums[i] == -999)
        {
            nums[i] = 0;
            break;
        }
        min_cst = min(min_cst, nums[i]);
        max_cst = max(max_cst, num[i]);
    }

    cout << max_cst << ' ' << min_cst;
    return 0;
}